src/components/button/primary.njk

```diff
@@ -0,0 +1,25 @@
+---
+layout: layout-example.njk
+---
+{% from "button/macro.njk" import govukButton %}
+
+<h3 class="govuk-heading-m">Input button</h3>
+
+{{ govukButton({
+  "text": "Save and continue"
+}) }}
+
+<h3 class="govuk-heading-m">Link button</h3>
+
+{{ govukButton({
+  "text": "Link button",
+  "href": "/"
```

Comment:
Only a small thing but instead of `/` can we use `#` as currently clicking on the button opens the homepage which is a bit weird.

Reply:
Fixed